resources/run_adf.sh

```diff
@@ -0,0 +1,319 @@
+#!/bin/bash
+# 
+# Usage: run_adf.sh <operation>
+# operation: 'start' or 'stop'
+#
+# This script will start a local mongod and Atlas Data instance, used for integration testing.
```

Comment:
I think that you wanted to update the naming, isn't it?
```suggestion
# This script will start a local mongod and Atlas Data Federation instance, used for integration testing.
```

Reply:
Good catch, thanks 👍